### PR TITLE
Templates and translations

### PR DESCRIPTION
**Pull request by okaycj**
# Summary

This PR moves two templates and translations to a new "Templates" package.  This new package is the location of templates and their translations moving forward. 

# For Another PR

* Add the ability to select which version of the template is needed.

### CODE REVIEW

File: packages/record/src/consentVideo.spec.ts [hunk 1]
```diff
@@ -179,10 +182,12 @@ test("recordButton", async () => {
...
   const plugin = new VideoConsentPlugin(jsPsych);
   const display = document.createElement("div");
+  const trial = { locale: "en-us" } as unknown as TrialType<PluginInfo>;
+
+  display.innerHTML = chsTemplates.consentVideo(trial);
 
   display.innerHTML =
-    Handlebars.compile(consentVideoTrial)({}) +
-    Handlebars.compile(recordFeed)({});
+    chsTemplates.consentVideo(trial) + Handlebars.compile(recordFeed)({});
```
Comment (becky-gilbert) [inline packages/record/src/consentVideo.spec.ts:190]: Ah. So we should keep using `Handlebars.compile` if the template has no translations?

File: packages/style/package.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
     "test": ""
   },
   "devDependencies": {
-    "@jspsych/config": "^3.0.0",
+    "@jspsych/config": "^2.0.0",
```
Comment (becky-gilbert) [inline packages/style/package.json:20]: Just curious why you bumped this down to 2?

File: packages/record/hbs/video-config.hbs
Comment (becky-gilbert) [inline packages/record/hbs/video-config.hbs:1]: Just to confirm: it looks like any templates in `record` that contain to-be-translated text (e.g. video-config) need to be moved to `templates`, but any templates that don't need translations (e.g. webcam-feed) can stay in `record`. Is that right?

**Review by becky-gilbert — APPROVED** [review]
Looks great!! I love the way this turned out - very nice and user friendly.

Approved pending passing checks 😄 

I'll start a PR to move the video-consent template/translations into `templates`.

File: packages/record/src/consentVideo.spec.ts (hunk 1 above)
Comment (okaycj) [reply packages/record/src/consentVideo.spec.ts:190]: That seems correct for now.  I am not sure how this is all going to shake out. Do you have a preference in this space?

File: packages/style/package.json (hunk 2 above)
Comment (okaycj) [reply packages/style/package.json:20]: v3 breaks builds.  I am going to fix this after we get the alpha release out.  I might not make it to the weekend.

File: packages/record/hbs/video-config.hbs
Comment (okaycj) [reply packages/record/hbs/video-config.hbs:1]: Yes, but up to a point.  I don't want things to get confusing or require too many rules.  What is easiest is most important to me.

File: packages/record/src/consentVideo.spec.ts (hunk 1 above)
Comment (becky-gilbert) [reply packages/record/src/consentVideo.spec.ts:190]: No preference!

File: packages/record/hbs/video-config.hbs
Comment (becky-gilbert) [reply packages/record/hbs/video-config.hbs:1]: I don't think that's confusing, just wanted to check (but I'm also fine with all templates in one place, if we decide that's easier).

**Pull request merged**